controllers/operator/configmap.go

```diff
@@ -925,7 +925,7 @@ func isPublicCloud(client client.Client, namespace string, configMap string) boo
 	currentConfigMap := &corev1.ConfigMap{}
 	err := client.Get(context.TODO(), types.NamespacedName{Name: configMap, Namespace: namespace}, currentConfigMap)
 	if err != nil {
-		log.Info("Error getting configmap", configMap)
+		log.Info("Error getting configmap", " Name ", configMap)
```

Comment:
Could we change the message to something like the following? Assuming we're refraining from making more significant behavioral changes around handling IsNotFound vs. other errors, etc.
```suggestion
		log.Info("Unable to get ConfigMap", "ConfigMap.Name", configMap, "ConfigMap.Namespace", namespace)
```

Reply:
done